packages/test/src/test-time.ts

```diff
@@ -1,4 +1,5 @@
 import test from 'ava';
+// eslint-disable-next-line import/no-named-as-default
 import Long from 'long';
```

Comment:
This can be `import * Long from 'long'`, or you can enable `esModuleInterop` instead.

Reply:
`import * as Long` results in:

<img width="737" alt="image" src="https://user-images.githubusercontent.com/251288/181633264-44fd01cb-19f6-45de-bbb9-745e9a94e35b.png">

`esModuleInterop` results in:

<img width="949" alt="image" src="https://user-images.githubusercontent.com/251288/181636284-f690ecc2-d1fa-4f83-8b17-5d145c20d554.png">

Reply:
That seems a bug from plugin-import that it doesn't handle class static properties correctly.

Reply:
But on the other hand, `long` does not use `exports` field with `types` correctly.

https://unpkg.com/browse/long@5.2.0/package.json